parallel build is broken in v1.9.4

`make -j` gives:
```
Creating... KIM_Config file..... examples/Makefile.KIM_InstalledConfig.

*******************************************************************************

*******************************************************************************

*******************************************************************************
** Building... kim-api-objects                                               **

*******************************************************************************
** Building... kim-api-libs                                                  **
** Building... src/utils                                                     **
** Building... completion                                                    **
*******************************************************************************
*******************************************************************************
*******************************************************************************
*******************************************************************************
Creating... KIM_API_Version.h.
make[4]: *** No rule to make target 'libkim-api-v1.9.4-git+bc49ce611.dirty.GCC.linux.64bit.dynamic-load.so', needed by 'kim-api-v1-descriptor-file-match'.  Stop.
make[4]: *** Waiting for unfinished jobs....
Creating... descriptor-file-match.o.
Creating... intermediate file... kim_str.c.
Creating... KIM_API_Version.h.
Creating... KIM_API_F03_Multiples.incl.
```

`make -j1` works fine.

--- a/Makefile
+++ b/Makefile
@@ -40,6 +40,9 @@ include Makefile.KIM_Config
 #
 .PHONY: help
 
+# see https://github.com/openkim/kim-api/issues/4
+.NOTPARALLEL:
+
 help:
 	@printf "TARGETS FOR HELP\n"
 	@printf "'help'                       -- print this list of targets\n"

